### PR TITLE
Some fixes for v0.6.0

### PR DESCRIPTION
**Pull request by AlexLarra**

### CODE REVIEW

**Comment by markets** [comment]
hey @AlexLarra let's try with `Maily.init!` just before iterating over the mailers (inside the generator). Thanks!

File: lib/maily.rb [hunk 1]
```diff
@@ -29,7 +31,7 @@ def load_emails_and_hooks
       end
 
       # Load hooks
-      hooks_file_path = File.join(Rails.root, hooks_path)
+      hooks_file_path = File.join(Rails.root, hooks_path || DEFAULT_HOOKS_PATH)
```
Comment (markets) [inline lib/maily.rb]: hey @AlexLarra let's try with Maily.init! just before iterating over the mailers (inside the generator). Thanks!

File: app/controllers/maily/emails_controller.rb [hunk 2]
```diff
@@ -10,7 +10,13 @@ def index
 
     def show
       if !@maily_email.correct_number_of_arguments?
-        alert = "#{@maily_email.required_arguments.size} arguments needed for #{@maily_email.name} email"
+        alert = if @maily_email.optional_arguments.size > 0
+          "#{@maily_email.name} email requires #{@maily_email.required_arguments.size} \
+          to #{@maily_email.required_arguments.size + @maily_email.optional_arguments.size} arguments"
+        else
+          "#{@maily_email.required_arguments.size} arguments needed for #{@maily_email.name} email"
+        end
```
Comment (jrdi) [inline app/controllers/maily/emails_controller.rb]: Un poco cutre, no?

Comment (markets) [inline app/controllers/maily/emails_controller.rb]: oooo! @jrdi is in da house. What do you propose?

Comment (jrdi) [inline app/controllers/maily/emails_controller.rb]: :trollface:

Comment (AlexLarra) [inline app/controllers/maily/emails_controller.rb]: @jrdi enséñame a hacerlo mejor. Lo único que me enseñaste es a huír de JS y no se me da bien.

**Pull request merged**